src/main/java/uk/gov/hmcts/darts/addaudio/validator/AddAudioFileValidator.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.darts.addaudio.validator;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.tika.Tika;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.darts.common.exceptions.DartsValidationException;
+import uk.gov.hmcts.darts.common.util.Validator;
+import uk.gov.hmcts.darts.config.AllowedMediaConfig;
+import uk.gov.hmcts.darts.ws.CodeAndMessage;
+
+import java.io.IOException;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class AddAudioFileValidator implements Validator<MultipartFile> {
+
+    @Autowired
+    private final AllowedMediaConfig allowedMediaConfig;
+
+    public void validate(MultipartFile addAudioFileRequest) {
+        if (addAudioFileRequest.getSize() <= 0) {
+            throw new DartsValidationException(CodeAndMessage.ERROR);
```

Comment:
In all these validations I'd suggest we either add a more explicative message to the thrown error, or we log 
an error message that explains why validation is failing.
This would help issue investigation

Reply:
logging messages added